public/js/actions/index.js

```diff
@@ -24,7 +22,7 @@ function navigate() {
   };
 }
 
-module.exports = Object.assign(
-  { willNavigate, navigate },
+module.exports = (Object.assign(
+  ({ willNavigate, navigate }: any),
   breakpoints, eventListeners, sources, tabs, pause
-);
+) : typeof breakpoints);
```

Comment:
It statically gives the type system whatever type `breakpoints` is. This tells the system that what I'm returning has the exact structure of the `breakpoints` object, so it lets you do this dynamic `Object.assign` munging but still type it.